test/verify/check-example

```diff
@@ -28,6 +28,7 @@ from testlib import *
 @skipPackage("example")
 class TestExample(MachineCase):
 
+    @unittest.expectedFailure
```

Comment:
Please split out this change and un-blacklisting check-examples to a separate commit.